src/srp.toit

```diff
@@ -0,0 +1,127 @@
+// Copyright (C) 2023 Toitware ApS.
```

Comment:
Should this library be made a package?

Reply:
You mean to make this as a seperated package? But this one lacks of some functions from full SRP.

Reply:
We can leave it here for now.
SRP sounds like a functionality that could be useful in other contexts, but no need to make it generic already now.